### ISSUE TITLE
Azure function running on consumption plan, not triggered by blob events

### ISSUE THREAD

**Issue by jeevan90** [Azure/Azure-Functions#1022]
I have a azure function created with ARM Template by using powershell.

Function is blobtrigger type function running on **consumption plan**, to copy blob from source storage to destination storage.

When I upload blob to source storage it is not copied to destination. That means function is not executed.

When I browse function app through portal, function get invoked and do the required things as expected. Thereafter it works fine. It only happens when function app initially deployed by powershell script using ARM templates.

When I use App service plan instead of Consumption plan, everything works fine since function app is always running.

So I guess Issue is, when I create function app using ARM template and deployed using powershell it is in idle mode, and never triggered by blob events. Is my assumption correct or could you please help me to find the issue.  Thanks

**Comment by ColbyTresness** [comment]
I don't think that's the cause of this issue. @brettsam this could be a logging/monitoring issue, thoughts?

**Comment by brettsam** [comment]
This is likely a sync triggers issue.

@jeevan90 -- this was from a while ago but do you happen to know the powershell command you ran to deploy this? I'm wondering if it was using the /zip api, which doesn't sync triggers.

**Comment by skyline9002** [comment]
Yes, same situation here. Why consumption plan will go idle if the function not invoke for a while. So strange.

**Comment by brettsam** [comment]
@skyline9002 -- Do you know how you are deploying your app? If not, can you [share your function app name with us (privately)](https://github.com/Azure/azure-functions-host/wiki/Sharing-Your-Function-App-name-privately)?

**Comment by skyline9002** [comment]
Hi Brett,

I used visual studio to deploy the app. And I think I ticked the zip
option.

Below is the function id

Id=dbc97680-e2aa-4b80-8d79-6ce2581ba118

Id=aea11b1a-74e4-4fc5-a99f-3135178a0193


Both functions are triggered by the blob storage.

Thanks

Simon

**Comment by brettsam** [comment]
Can you try clicking the refresh button for your function in the portal? I'm not sure what the issue could be yet, but I want to see if that fixes it:
